### DISCUSSION TITLE
Picture Blur before loading

### DISCUSSION THREAD

**Discussion by nisaeggu** [silveredgold/beta-protection#24] [Ideas]
It would be really nice to have some sort of automatic blur option for the loadup of pictures like the videos have. When loading most websites, pictures are visible for a short time (even with placeholder) until beta protection kicks in. 

Something like this would be cool:
- Blur Pictures before placeholder is loaded
- Block Pictures before placeholder is loaded
- Blur Pictures before censor is finished
- Block Pictures before censor is finished

**Comment by silveredgold** [comment]
This is now available in 0.2.6 and later which blurs images before the placeholders replace the images while censoring takes place.
